packaging/cloudify-mgmtworker.spec

```diff
@@ -2,6 +2,12 @@
 %define __jar_repack %{nil}
 %define PIP_INSTALL /opt/mgmtworker/env/bin/pip install -c "${RPM_SOURCE_DIR}/packaging/mgmtworker/constraints.txt"
 
+# this prevents networkx<2 failure in RH8
```

Comment:
Do we need to make it conditional? If it'll work with RH/C 7 as well then maybe let's just do it.

Also, like @tehasdf said above, it'd be good to put a comment in saying what failures it is sidestepping (in the spec itself).

Reply:
I'm not positive that RH7 build even has `__brp_mangle_shebangs`, but I can't confirm it